smart-orchestrator-test/src_test/ch/ivyteam/test/log/ResourceResponse.java

```diff
@@ -0,0 +1,34 @@
+package ch.ivyteam.test.log;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.ws.rs.core.Response;
+
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+public class ResourceResponse implements BeforeEachCallback {
```

Comment:
very simple generalization of our resource-loading/response pattern. 
although small, it should reduce the amount of duplicated resource-loading related code in our test base.